BUILDING.md

```diff
@@ -35,7 +35,7 @@ file a new issue.
     * [Building Node.js](#building-nodejs-1)
```

Comment:
there are a few places in the docs that refer to full-icu not being the default (https://nodejs.org/dist/latest-v12.x/docs/api/util.html#util_whatwg_supported_encodings). Would be good to get those fixed up in a separate commit here.

Reply:
@jasnell  d4c101900622dc68adc73280d589ae66aa36580b PTAL

Reply:
Thank you!